11_hexcode/style.scss

```diff
@@ -0,0 +1,49 @@
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
+    height: 100%;
+}
+
+body {
+    font-size: 1.4rem;
+    background-color: #ffc0cb;
+}
+
+h1 {
+    font-size: 2.4rem;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+}
+
+.btn-name {
+    margin: 5rem auto;
+    width: 30rem;
+    text-align: center;
+}
+
+button {
+    margin: 0 auto;
+    width: 10rem;
+    height: 5rem;
+    font-size: larger;
+    border-radius: 10px;
+    background-color: transparent;
+    border: 2px solid rgba(0, 0, 0, 0.7);
+    color: rgba(0, 0, 0, 0.7);
+    transition: 0.7s;
+    outline: 0;
+}
+
+button:hover,
+button:focus {
+    transform: scale(1.5, 0.8);
```

Comment:
You really like to scale buttons, don't you? ehehe.
IRL try to avoid scaling text not proportionally to avoid difficulty in reading the text :)